media-sound/deadbeef/files/deadbeef-musl.patch

```diff
@@ -0,0 +1,25 @@
+Bug: https://bugs.gentoo.org/870187
```

Comment:
I like to include explicit links to the commit as well: https://github.com/DeaDBeeF-Player/deadbeef/commit/6864a5779fd6beb21507357736e0c988748cec5a

I'll throw that in on merge.

Reply:
Good catch, I forgot that one.